### PR TITLE
Package mem_usage.0.1.1

### PR DESCRIPTION
**Pull request by toots**
### `mem_usage.0.1.1`
Cross-platform stats about memory usage

---
* Homepage: https://github.com/savonet/ocaml-mem_usage
* Source repo: git+https://github.com/savonet/ocaml-mem_usage.git
* Bug tracker: https://github.com/savonet/ocaml-mem_usage/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by samoht** [comment]
```
2024-04-17 20:26.07: Error in mem_usage.0.1.1:            warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/ : "GPL-2.0"
2024-04-17 20:26.19: Job failed: Error in mem_usage.0.1.1:            warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/ : "GPL-2.0"
```

File: packages/mem_usage/mem_usage.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Cross-platform stats about memory usage"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "GPL-2.0"
```
Comment (samoht) [inline packages/mem_usage/mem_usage.0.1.1/opam:6]: I guess this should be `GPL-2.0-only` or `GPL-2.0-or-after`

Comment (toots) [reply packages/mem_usage/mem_usage.0.1.1/opam:6]: It's actually MIT. Fixed upstream, thanks for spotting!

Comment (toots) [inline packages/mem_usage/mem_usage.0.1.1/opam:6]:
```suggestion
license: "MIT"
```

**Comment by samoht** [comment]
The packae fails to compile on FreeBSD - could you add a constraint in the opam file? I'm happy to merge it anyway as FreeBSD is an experimental target for now.

**Comment by toots** [comment]
> The packae fails to compile on FreeBSD - could you add a constraint in the opam file? I'm happy to merge it anyway as FreeBSD is an experimental target for now.

Thanks. I have made the corresponding changes upstream.

**Pull request merged**